recipes-graphics/spark/spark_git.bb

```diff
@@ -0,0 +1,100 @@
+    -DBUILD_WITH_TEXTURE_USAGE_MONITORING=ON \
+    -DPXCORE_COMPILE_WARNINGS_AS_ERRORS=OFF \
+    -DPXSCENE_COMPILE_WARNINGS_AS_ERRORS=OFF \
+    -DCMAKE_SKIP_RPATH=ON \
+    -DPXCORE_MATRIX_HELPERS=OFF \
+    -DBUILD_PXWAYLAND_SHARED_LIB=OFF \
+    -DBUILD_PXWAYLAND_STATIC_LIB=OFF \
+    -DPREFER_SYSTEM_LIBRARIES=ON \
+    -DDISABLE_TURBO_JPEG=ON \
+    -DDISABLE_DEBUG_MODE=ON \
+    -DSPARK_BACKGROUND_TEXTURE_CREATION=OFF \
+    -DSPARK_ENABLE_LRU_TEXTURE_EJECTION=OFF \
+    -DSUPPORT_DUKTAPE=OFF \
+    -DBUILD_DUKTAPE=ON \
+    -DBUILD_PXSCENE_APP=OFF \
+    -DBUILD_PXSCENE_STATIC_LIB=OFF \
+    -DBUILD_PXSCENE_SHARED_LIB=ON \
+    -DBUILD_PXSCENE_APP_WITH_PXSCENE_LIB=ON \
+    -DBUILD_RTCORE_LIBS=ON \
+    -DBUILD_RTCORE_STATIC_LIB=OFF \
+    -DSPARK_ENABLE_OPTIMIZED_UPDATE=OFF \
+"
+TARGET_CXXFLAGS += " -fno-delete-null-pointer-checks "
+
+do_install() {
+    install -d ${D}${includedir}
+    cp -ar ${S}/src/*.h ${D}${includedir}
+    install -d ${D}${includedir}/unix
+    cp -Rpf ${S}/src/unix/*.h ${D}${includedir}/unix
+
+    install -d ${D}${includedir}/spark
+    cp -ar ${S}/examples/pxScene2d/src/*.h ${D}${includedir}/spark
+
+    install -d ${D}${includedir}/spark/${COMPOSITOR}
+    cp -ar ${S}/src/${COMPOSITOR}/*.h ${D}${includedir}/spark/${COMPOSITOR}
+
+    install -d ${D}${libdir}/pkgconfig
+    install -m 644 ${WORKDIR}/Spark.pc ${D}${libdir}/pkgconfig/Spark.pc
+
+    install -d ${D}${libdir}
+    install -m 755 ${S}/examples/pxScene2d/src/libSpark.so ${D}${libdir}
+    install -m 755 ${S}/build/${LIBRTCORE_SUBDIR}/librtCore.so ${D}${libdir}
+
+    install -d ${D}${datadir}/WPEFramework/Spark
+    cp -av --no-preserve=ownership ${S}/examples/pxScene2d/src/node_modules ${D}${datadir}/WPEFramework/Spark/
+    install -m 755 ${S}/examples/pxScene2d/src/*.js ${D}${datadir}/WPEFramework/Spark/
+    install -m 755 ${S}/examples/pxScene2d/src/*.json ${D}${datadir}/WPEFramework/Spark/
+    install -m 755 ${S}/examples/pxScene2d/src/*.ttf ${D}${datadir}/WPEFramework/Spark/
+    install -m 755 ${S}/examples/pxScene2d/src/sparkpermissions.conf ${D}${datadir}/WPEFramework/Spark/
+    install -m 755 ${S}/examples/pxScene2d/src/waylandregistry.conf ${D}${datadir}/WPEFramework/Spark/
+    cp -av --no-preserve=ownership ${S}/examples/pxScene2d/src/rcvrcore ${D}${datadir}/WPEFramework/Spark/
+    cp -av --no-preserve=ownership ${S}/examples/pxScene2d/src/browser ${D}${datadir}/WPEFramework/Spark/
+    cp -av --no-preserve=ownership ${S}/examples/pxScene2d/src/optimus ${D}${datadir}/WPEFramework/Spark/
+    cp -av --no-preserve=ownership ${S}/examples/pxScene2d/src/duk_modules ${D}${datadir}/WPEFramework/Spark/
+    cp -av --no-preserve=ownership ${S}/examples/pxScene2d/src/v8_modules ${D}${datadir}/WPEFramework/Spark/
+    cp -av --no-preserve=ownership ${S}/examples/pxScene2d/src/rasterizer ${D}${datadir}/WPEFramework/Spark/
+}
+
+# ----------------------------------------------------------------------------
+
+FILES_SOLIBSDEV = ""
```

Comment:
Is this needed?

Reply:
removing  this endup with QA Issue: -dev package spark-dev contains non-symlink .so, if add INSANE_SKIP_${PN}-dev = "dev-elf" to avoid it will again make issue for wpeframework-plugins as  QA Issue: wpeframework-plugins rdepends on spark-dev [dev-deps]